public/locales/pl/translation.json

```diff
@@ -0,0 +1,44 @@
+{
+  "common": {
+    "logoAlt": "Logo aplikacji",
+    "scrollIconTitle": "Ikona przewijania strony do początku",
+    "downloadIcon": "Ikona pobierania",
```

Comment:
Could we make it consistent, so either always use only Icon not adding "title" "label" "aria"? Or are those cases somehow different?

Reply:
You're right, I've removed "title".